integreatly/1.1.0/integreatly-operator.1.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,243 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: integreatlyoperator.1.0.1
```

Comment:
this needs to change too

Reply:
Yeah I had to change it back due to an error from olm about 2 csvs. Doing this got it working

Reply:
moved back to 1.0.1 but updated the image